src/components/LandingPage.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
```

Comment:
Thank you for adding a landing page! As with other of my comments in this PR, I think this is a great addition for volunteer functionality.  However when we go public, this will be confusing as general users will not be using the app as a "timestamping tool" but as more of a "youtube for mbt" where they can search, like videos, create playlists, etc.